<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewCompany</class>
 <widget class="QDialog" name="addNewCompany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QWidget" name="addNew">
    <widget class="QLineEdit" name="addNew_cityStateZip">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="addNew_websiteName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="addNew_companyName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Company Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Website Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addNew_streetName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Street Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>City, State, Zip:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addNew_submitButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>440</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QComboBox" name="addNew_keyCustomerComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>380</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>325</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Interest Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>385</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Key Customer:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="addNew_interestLevelComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>320</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="editCurrent">
    <widget class="QPushButton" name="editCurrent_submitButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>460</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Street Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Company Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>415</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Key Customer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Website Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editCurrent_companyName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>355</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Interest Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>City, State, Zip:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editCurrent_streetName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editCurrent_websiteName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editCurrent_cityStateZip">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editCurrent_currentCompany">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Currently Selected Company:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="editCurrent_interestLevelComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="editCurrent_keyCustomerComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>410</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
